rst_files/getting_started.rst

```diff
@@ -336,38 +303,6 @@ which provides a static HTML representations of notebooks
 
 QuantEcon also hosts the `QuantEcon Notes <http://notes.quantecon.org/>`_ website, where you can upload and share your notebooks with other economists and the QuantEcon community
 
-.. _julia_repl: 
-
-The REPL
-------------
-
-While we have not emphasized it, on any :ref:`JupyterHub <jl_jupyterhub>` or local Jupyter 
-installation you will also have access to the Julia REPL
-
-This is a Julia specific terminal disconnected from the graphical interface of 
-Jupyter, and becomes increasingly important as you learn Julia
-
-For example, the REPL is what we used in the beginning of this lecture to install 
-``InstantiateFromURL.jl`` and ``IJulia``
-
-To start the REPL in a typical Jupyter lab environment
-
-#. Choose "New Launcher"
-#. Choose a ``Julia 1.0`` Console
```

Comment:
This stuff is not obvious to people, and is sometimes necessary.  Again, the issue is that all notebook and REPL based stuff has been moved out of the getting started otherwise.

But perhaps it could be condensed a little.

Reply:
It ended up being frequently necessary in the course we taught.  There is no other place in the lectures for this sort of stuff since we tell people that the "Tools and Editors" is optional.  But the section on the REPL can be marked `advanced` so that people can refer back to it when they need it.

One option is to tell people how to launch the REPL and then point them at the https://lectures.quantecon.org/jl/tools_editors.html# sections for doing a few operations.  But I am hesitant to say we have given people the tools to use julia without telling them that they can (and sometimes should) play in the REPL.  This section has already been significantly reduced from the older version of the notes.